### PR TITLE
MF-524: Fix vitals signs display values when units are absent

### PR DESCRIPTION
**Pull request by denniskigen**
Carried over from https://github.com/openmrs/openmrs-esm-patient-chart-widgets/pull/232.

### CODE REVIEW

File: packages/esm-patient-vitals-app/src/vitals/vitals-overview.test.outdated.tsx [hunk 1]
```diff
@@ -1,54 +1,107 @@
 import React from "react";
-import VitalsOverview from "./vitals-overview.component";
 import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 import { BrowserRouter } from "react-router-dom";
 import { of } from "rxjs/internal/observable/of";
-import { mockVitalData } from "../../../../__mocks__/vitals.mock";
+import { mockVitalData } from "../../__mocks__/vitals.mock";
```
Comment (FlorianRappl) [inline packages/esm-patient-vitals-app/src/vitals/vitals-overview.test.outdated.tsx:6]: Well, all mock should be aggregated in the top-level mocks dir. Can you change this?

File: packages/esm-patient-vitals-app/src/vitals/vitals-overview.component.tsx [hunk 2]
```diff
@@ -55,203 +49,198 @@ const RenderVitals: React.FC<RenderVitalsProps> = ({
     ,
     ,
     pulseUnit,
-    oxygenationUnit,
+    oxygenSaturationUnit,
     ,
     respiratoryRateUnit,
   ] = conceptsUnits;
 
+  const [isLoadingPatient, , patientUuid] = useCurrentPatient();
```
Comment (FlorianRappl) [inline packages/esm-patient-vitals-app/src/vitals/vitals-overview.component.tsx:57]: Don't use this.

**Review by FlorianRappl — CHANGES_REQUESTED** [review]

File: packages/esm-patient-vitals-app/src/vitals/vitals-overview.component.tsx (hunk 2 above)
Comment (denniskigen) [reply packages/esm-patient-vitals-app/src/vitals/vitals-overview.component.tsx]: Fixed. Did a bad rebase.

File: packages/esm-patient-vitals-app/src/vitals/vitals-overview.test.outdated.tsx (hunk 1 above)
Comment (denniskigen) [reply packages/esm-patient-vitals-app/src/vitals/vitals-overview.test.outdated.tsx]: Fixed

**Review by FlorianRappl — APPROVED** [review]

**Pull request merged**